.azure/pipelines/ci.yml

```diff
@@ -60,4 +60,14 @@ jobs:
   parameters:
     TestGroupName: IISBackCompat
     skipArgs: "/p:SkipIISBackwardsCompatibilityTests=false /p:SkipIISTests=true /p:SkipIISExpressTests=true /p:SkipIISForwardsCompatibilityTests=true"
-- template: jobs/site-extensions-job.yml
\ No newline at end of file
+- template: jobs/site-extensions-job.yml
+- job: Helix
```

Comment:
Are you going to remove this from the main config too?

Reply:
Yep, removed